### PR TITLE
Chore: Delete project cache before using in a test

### PR DESCRIPTION
**Pull request by crericha**

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -131,3 +134,11 @@ def _make_function(model: Model, version: t.Optional[str] = None, **kwargs) -> S
 @pytest.fixture
 def random_name() -> t.Callable:
     return lambda: f"generated_{random_id()}"
+
+
+def delete_cache(project_paths: str | t.List[str]) -> None:
```
Comment (tobymao) [inline tests/conftest.py:139]: is there no method on cache to delet?

Comment (izeigerman) [reply tests/conftest.py:139]: There's currently none. And I'm not sure we need one outside tests

File: tests/dbt/conftest.py [hunk 2]
```diff
@@ -11,7 +12,12 @@
 
 @pytest.fixture()
 def sushi_test_project(mocker: MockerFixture) -> Project:
-    project = Project.load(DbtContext(project_root=Path("tests/fixtures/dbt/sushi_test")))
+    project_root = Path("tests/fixtures/dbt/sushi_test")
+    try:
+        rmtree(str(project_root) + "/.cache")
```
[the remark this answers is not in the document]
Comment (crericha) [reply tests/dbt/conftest.py]: Addressed

File: tests/conftest.py (hunk 1 above)
Comment (crericha) [reply tests/conftest.py:139]: What's the verdict? Good as is or this should be built into the cache code?

Comment (crericha) [reply tests/conftest.py:139]: @tobymao

**Review by tobymao — APPROVED** [review]

**Pull request merged**